javascript.html.markdown

```diff
@@ -603,31 +603,39 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement without '{ }' defined will be the return statement, hence the `return` keyword is not required. 
 
-const isEven = (number) => {
-    return number % 2 === 0;
-};
+const isOdd = number => number % 2 !== 0;
+
+// But, if '{ }' are used, then a `return` is required.
 
-isEven(7); // false
+const isEven = number => { return number % 2 == 0 }
 
 // The "equivalent" of this function in the traditional syntax would look like this:
 
-function isEven(number) {
-    return number % 2 === 0;
+function isOdd(number) {
+    return number % 2 !== 0;
 };
 
-// I put the word "equivalent" in double quotes because a function defined
-// using the lambda syntax cannnot be called before the definition.
+// However if a function has no arguments, a slight change of adding the parentheses in it's place is required:
+const helloWorld = () => {
+    console.log("Hello World")
+}
+
+// "Equivalent" refers to the fact that a function defined
+// using the arrow function cannnot be called before the definition.
```

Comment:
To me this seems like "equivalent" is a meaningful word here, where it's actually not.

Instead I'd move this information to the part where we make the distinction between function declarations and function expressions and say something like:

> A function declaration _can_ be called before it's defined in a JavaScript file (this is a result of "hoisting").

```
add(1,8);

function add(number1, number2) {
  return number1 + number2;
}
```
> On the other hand a function expression _cannot_ be called before it's defined.

```
add(1,8); // error!

const add = function (number1, number2) {
  return number1 + number2;
};
```




```

```